build_tools/benchmarks/comparisons/setup_desktop.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
```

Comment:
Nit: license header.

You might also want `set -euo pipefail` or some subset thereof

Reply:
Done